packages/integration/src/helpers.test.ts

```diff
@@ -88,6 +88,24 @@ describe('isValidHost', () => {
 
 describe('defaultScmResolveUrl', () => {
   it('works for relative paths and retains query params', () => {
+    expect(
+      defaultScmResolveUrl({
+        url: './b.yaml',
+        base: 'https://gitlab.com/groupA/teams/teamA/subgroupA/repoA/-/blob/branch/folder/',
+      }),
+    ).toBe(
+      'https://gitlab.com/groupA/teams/teamA/subgroupA/repoA/-/blob/branch/folder/b.yaml',
+    );
+
+    expect(
```

Comment:
Note the difference in behavior between this and the above test case. If you resolve `./b.yaml` against `<repoRoot>/folder/` you get `<repoRoot>/folder/b.yaml`, but if you resolve it against `<repoRoot>/folder` with no trailing slash, you get `<repoRoot>/b.yaml`. I think this makes sense, but want to call it out in case it merits some discussion.